khorne.go

```diff
@@ -29,8 +31,33 @@ type Result struct {
 	Error   string `json:"error,omitempty"`
 }
 
-func (p *Chaos) RunChaosExperiment(namespace string, configPath string) Result {
-	if err := chaos.RunChaosExperiment(namespace, configPath); err != nil {
+func (p *Chaos) RunChaosExperiment(call goja.FunctionCall, runtime *goja.Runtime) goja.Value {
+	namespace := call.Argument(0).String()
+	configObj := call.Argument(1).Export()
+
+	manifestObj, ok := configObj.(map[string]interface{})
+	if !ok {
+		panic("Invalid manifest object")
+	}
+	manifestStruct := &unstructured.Unstructured{
+		Object: manifestObj,
+	}
+
+	response := map[string]interface{}{
+		"success": true,
+		"error":   "",
+	}
+	if err := chaos.RunChaosExperiment(namespace, manifestStruct); err != nil {
+		response["success"] = false
+		response["error"] = err.Error()
+		return runtime.ToValue(response)
+	}
+
+	return runtime.ToValue(response)
```

Comment:
This code is structured strangely. Here it would be better to form an error response structure in case of error and a successful structure in case of success.

```suggestion
	if err := chaos.RunChaosExperiment(namespace, manifestStruct); err != nil {
		return runtime.ToValue(map[string]interface{}{"success": false, "error": err.Error()})
	}

	return runtime.ToValue(map[string]interface{}{"success": true, "error": ""})
```